examples/frontend/package.json

```diff
@@ -14,12 +14,12 @@
     "react-dom": "^18.0.0"
   },
   "devDependencies": {
-    "@openapi-codegen/cli": "^1.5.0",
-    "@openapi-codegen/typescript": "^3.0.1",
+    "@openapi-codegen/cli": "^1.5.2",
+    "@openapi-codegen/typescript": "^5.1.0",
     "@types/react": "^18.0.0",
     "@types/react-dom": "^18.0.0",
     "@vitejs/plugin-react": "^1.3.0",
-    "typescript": "^4.6.3",
+    "typescript": "^4.7.4",
```

Comment:
Any reason to not bump to `4.8.2`? 🤔

Reply:
just to test issue #88 with it. It was in the example folder so I left it like this for now. I can bump it, no problem.

Reply:
Yeah, let's bump it! I will test locally with [verdaccio](https://verdaccio.org/) to see if everything is following

Reply:
bumped 👍